app/src/main/res/values/string_array_items.xml

```diff
@@ -10,7 +10,8 @@
     <string name="seasons_title_summer">summer</string>
     <string name="seasons_title_fall">fall</string>
 
-    <string name="trending_title_being_watched">being watched</string>
+    <string name="trending_title_anime">trending anime</string>
+    <string name="trending_title_manga">trending manga</string>
```

Comment:
We'll need to update all the `string_array_items` in the other resource folders too 😃

Reply:
Done.